src/api/app/controllers/public_controller.rb

```diff
@@ -228,4 +192,39 @@ def unshift_public(path)
       return path
     end
   end
+
+  def extract_user_public
+    # to become _public_ special user
+    if ::Configuration.anonymous
+      load_nobody
+      return true
+    end
+    logger.error 'No public access is configured'
+    render_error( :message => 'No public access is configured', :status => 401 )
+    return false
```

Comment:
Actually, in a controller one would just call return, or don't have it at all.

But as Christian said, he was just moving the code around